.circleci/config.yml

```diff
@@ -125,7 +125,7 @@ jobs:
         type: string
         default: 'oplabs-tools-artifacts/internal-images'
     machine:
-      image: ubuntu-2204:2022.07.1
+      image: ubuntu-2204:current
```

Comment:
one thing to watch out for is when we unpin ourselves from a specific version as this OS version gets updated builds can start failing randomly if deps ever get updated on the OS. I think in general it's always best to ping to a specific version

Reply:
good catch, pinned to 2024.01.01